cmd/minikube/cmd/cp.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"k8s.io/minikube/pkg/minikube/assets"
+	"k8s.io/minikube/pkg/minikube/exit"
+	"k8s.io/minikube/pkg/minikube/mustload"
+	"k8s.io/minikube/pkg/minikube/out"
+	"k8s.io/minikube/pkg/minikube/reason"
+)
+
+// placeholders for flag values
+var (
+	srcPath string
+	dstPath string
+)
+
+// cpCmd represents the cp command, similar to docker cp
+var cpCmd = &cobra.Command{
+	Use:   "cp <source file path> <target file path followed by '/home/docker/'>",
```

Comment:
I'm a bit confused by this description, I thought the command would work something like: 

```
minikube cp a.txt. /b.txt
```

Where does the /home/docker come in?

Additionally, we should probably insist that the second argument is an absolute path.

Reply:
I agree with you the prefix_path '/home/docker/' is confusing :)
By the way, when we do 'minikube ssh', the starting path is '/home/docker' since the default username is docker.
Is it ok to copy on '/somewhere' where the permission is only for 'root user' ?

I think there maybe 3 options: (or you have any idea, please let me know :) )
1) minikube cp a.txt /home/docker/b.txt
  - make minikube user to write '/home/docker' to see exactly where the path is.
  - But, they should write the /home/docker path for every copying
2) minikube cp a.txt /b.txt
  - enable copy at root permissioned directory
3) minikube cp a.txt b.txt
  - minikube user doesn't care about the user name, but always will be at /home/docker/b.txt as what i've done

Reply:
What Option would be the best ? @priyawadhwa 
In my opinion for now, option 1 or option 3 will be safe.

Reply:
I think we should ask the user for an absolute path and try to copy to wherever they want the file copied. If it fails because the command requires `sudo`, then we should provide a `--force` or `-f` flag which will copy over the file with root permissions.

Reply:
Thanks for your comment.
I've changed to minikube copy can copy anywhere in minikube as your opinion.
Could you re-check? @priyawadhwa